index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>project2</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Playfair+Display&family=Playfair+Display+SC&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+<body>
+  <!-- trying to make it so that the picture is to the right and text is laying over it-- referred to "clearfix" and "text on picture" articles on w3schools.com -->
+  <div class="card">
+    <img
+      src="https://cdn.glitch.com/7af50fa5-5455-4b82-8564-57e5e1ea17ac%2FThe%20Boys%20in%20the%20Band.jpg?v=1602748106231"
+      width="512"
+      height="308"
+    />
+
+    <div class="text">
+        <h2>The Boys in the Band</h2>
+
+        <nav>
+          <a href="#"> 2020</a>
+          <span class="divider">|</span>
+          <a href="#"> R</a>
+          <span class="divider">|</span>
+          <span class="time">2h2m</span>
+          <span class="divider">|</span>
+          <a href="#"> LGBTQ+ Movies,</a>
+          <a href="#"> Dramas</a>
+        </nav>
```

Comment:
Instead of a `nav`, I'd recommend an `ul` for these:
```suggestion
        <ul>
                <li>
                        <a href="#">2020</a>
                </li>
                <li>
                        <a href="#">R</a>
                </li>
                  <li class="time">2h2m</li>
                <li>
                        <a href="#">LGBTQ+ Movies</a>, <a href="#">Dramas</a>
                </li>
        </ul>
```

Doing things this way, you can also eliminate the divider `span`s and use `border-left` to recreate that line instead. In your CSS, it would look like this:

```css

/* Inline #2 | https://weichenhwang-gd2-project2.glitch.me/# */

ul {
  list-style: none;
  margin-top: 0;
  margin-bottom: 0;
  padding-left: 0;
  /* plus these styles from your nav */
  color: gray;
  font-family: "Open Sans", sans-serif;
  font-style: italic;
  font-size: 16px;
  word-spacing: 5px;
}

li {
  display: inline-block;
  line-height: 1.2;
}

li + li {
  border-left: 2px solid gray;
  margin-left: 6px;
  padding-left: 6px;
}
```